### DISCUSSION TITLE
Выбираем стиль значков для контрибьютеров Доки

### DISCUSSION THREAD

**Discussion by solarrust** [doka-guide/content#3476] [Идеи]
Привет! 🖖

У нас давно есть идея раздавать контрибьютерам — вам — классные значки за разные достижения. Для этого есть почти всё: мы умеем учитывать ваши действия в Доке, у нас есть отдельная страница для каждого контрибьютера, где значки будут классно смотреться. 
Дело за малым — выбрать дизайн!

Прямо сейчас есть два варианта. 

![image](https://user-images.githubusercontent.com/13676514/183082935-b81e93da-bbec-4d59-aed3-483e49b999aa.png)

Читайте ниже подробности 👇

## Первый вариант: текстовый

Тут в основе идеи лежат текстовые смайлики, набравшие популярность в Азии и распространённые в IT-сообществе. Эта идея хорошо вяжется с нашим символьным логотипом, Булочкой, U•ᴥ•U.

![telegram-cloud-photo-size-2-5413786198092135682-y](https://user-images.githubusercontent.com/13676514/182920716-792803ec-53b6-405f-8db8-15a91ac1bf93.jpg)

![telegram-cloud-photo-size-2-5413786198092135683-y](https://user-images.githubusercontent.com/13676514/182920758-d2985b06-161b-4e45-afe9-d37b4f447767.jpg)

## Вторая идея: графичная

В этом варианте взяты отдельные элементы из наших волшебных иллюстраций, подходящие по смыслу к действию, за которое даётся значок. Чуть меньше связи с лого Доки, но крепкая приемственность с большими и красивыми картинками.

![telegram-cloud-photo-size-2-5427339663179366721-y](https://user-images.githubusercontent.com/13676514/182921519-e33cff6c-edf0-47dd-a611-c54fb0659f33.jpg)

![telegram-cloud-photo-size-2-5427339663179366722-y](https://user-images.githubusercontent.com/13676514/182921539-01b30b53-0757-4f02-bebb-a93200c28e5a.jpg)

Мнения в редакции разделились и мы решили спросить у вас, какой вариант вам нравится больше. Вам эти значки _носить_ в конце концов =)

Высказывайте свои мнения в комментариях! Очень их ждём ♥️

**Comment by 2bj** [comment]
+1 for txt

**Reply by envyvox** [reply]
Я не контрибьютор, но если вдруг мнения простых смерных будут важны - определенно текстовые значки.

**Comment by cergmin** [comment]
+1 за графичные

**Comment by dmorenogogoleva** [comment]
+1 txt

**Comment by kravchuk21** [comment]
За текстовые, но поменять бы самые просматриваемые под стиль логотипа U•ᴥ•U
![IMG_20220805_162759](https://user-images.githubusercontent.com/66268773/183087751-42aea7cb-26d9-46ec-ab4b-5a7e997ef906.jpg)

**Comment by skrylnikov** [comment]
+1 за графические

**Comment by blackskip-notOk** [comment]
+1 графические

**Comment by sakraynev** [comment]
+1 текстовые

**Comment by imaxsof** [comment]
+1 за графичные, легче воспринимаются

**Comment by nlopin** [comment]
Мне нравятся текстовые значки. Они необычные и их интересно разглядывать. В то же время, не все они хорошо читаются, думаю это решится дополнительными итерациями и тултипами

**Comment by pa7lux** [comment]
Я за графичные, потому что могу легко в них считать образ. Текстовые похожи на минифицированный код: разобраться можно, но прилагая усилия. Мне комфортнее, если профиль не требует от меня ментальных упражнений, чтобы просто смотрел и думал "какой я классный"

**Comment by deathrollru** [comment]
Однозначно за текстовые. В них есть свой шарм.

**Comment by fpetrakov** [comment]
Аааа, какие они все классные! Но текстовые нравятся чуточку больше ( ͡° ͜ʖ ͡°)

**Comment by TatianaFokina** [comment]
Я за текстовые.

Проблему с читаемостью текстовых может решить подпись. Графические тоже, на самом деле, может быть сложно считать с ходу. Всё же тут не сложившиеся паттерны, а авторские штуки. Поэтому по сложности восприятия они на одном уровне, а вот по эффекту мимими побеждают текстовые :)

**Comment by igsekor** [comment]
Я за текстовые.

Кажется, мы вносим таким образом некий уникальный стиль.

**Comment by BlueInGreen68** [comment]
Текстовые!

**Comment by StarHamster** [comment]
+1 текстовые

А почему не воспользовались новой крутой фичей гитхаба — [опросами](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#creating-a-poll)?)
